### PR TITLE
Integrating yaydoc to Susi Server

### PR DESCRIPTION
**Pull request by imujjwal96**
Fixes issue #380 

### Changes
- Added the yaydoc configuration file
- Removed the conf.py file. It was overriding yaydoc's configurations.
- Updated the `docs/installation/installation_windows_generic.md` markdown fixing errors that caused bugs in rendering docs.

### Demo
Here are the generated docs with these changes: https://imujjwal96.github.io/susi_server/

__NOTE__: Please raise all the ui issues of the theme in bug tracker of sphinx_fossasia_theme (https://github.com/fossasia/sphinx_fossasia_theme/issues)

### Screenshot
image

### CODE REVIEW

**Comment by imujjwal96** [comment]
@mariobehling  @niranjan94 
@pri22296 @sch00lb0y 
Please review

**Comment by pri22296** [comment]
Links at https://imujjwal96.github.io/susi_server/#how-do-i-install-susi-ai-with-docker-on-google-cloud are not working due to the extension being .md instead of .html

File: .yaydoc.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+metadata:
```
Comment (pri22296) [inline .yaydoc.yml]: Update this file as per the new design of `.yaydoc.yml`

**Comment by mariobehling** [comment]
Why is this still marked WIP?

@imujjwal96 If you want reviews don't label PRs as WIP. Most people will not review a PR when it is labeled WIP.

**Comment by imujjwal96** [comment]
Generated docs: http://yaydoc.herokuapp.com/preview/ujjwalb1996@gmail.com/a6b3b99f-d104-4175-8f89-945c000fe959_preview/

**Comment by imujjwal96** [comment]
@pri22296  @sch00lb0y  Please review.

File: docs/index.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.. include:: README.rst
+.. include:: ../README.rst
```
Comment (pri22296) [inline docs/index.rst]: Add links to swagger docs and javadocs here?

Comment (imujjwal96) [inline docs/index.rst]: @pri22296 What's the link for javadocs?

Comment (sch00lb0y) [inline docs/index.rst]: @imujjwal96 hyper link like this `javadoc <./javadoc>`_  and also add java build pack to Heroku javadoc is not generating

Comment (imujjwal96) [inline docs/index.rst]: @pri22296  @sch00lb0y I added them but the toctree is not being generated: yaydoc.herokuapp.com/preview/ujjwalb1996@gmail.com/a3f37342-7bdd-4456-a7ef-de63b198218d_preview/

Comment (pri22296) [inline docs/index.rst]: Can you share a link again?

**Comment by mariobehling** [comment]
@imujjwal96 @pri22296 @sch00lb0y I am merging this as work in progress and as this is an improvement of the current docs. Please follow up. Thank you.

**Pull request merged**